navigation/navigation-common/src/commonTest/kotlin/androidx/navigation/serialization/NavArgumentGeneratorTest.kt

```diff
@@ -648,338 +648,6 @@ class NavArgumentGeneratorTest {
         assertThat(converted[0].argument.isDefaultValueUnknown).isFalse()
     }
 
-    @Test
-    fun convertToEnumList() {
```

Comment:
Is it moved to somewhere?

Reply:
Is it inside `android*Test` now or just deleted?

Reply:
yes. it was there already in our fork